tools/build.js

```diff
@@ -0,0 +1,24 @@
+import webpack from 'webpack';
+import webpackConfig from '../webpack.config.prod';
+
+process.NODE_ENV = 'production';
+
+console.log('Generatng minified bundle files via Webpack, This will take a moment');
```

Comment:
Why do have this console.log statement here?

Reply:
It's actually for production,  so this is just to know that webpack is generating minified bundle files